lib/python/qmk/keymap.py

```diff
@@ -130,9 +169,9 @@ def list_keymaps(keyboard_name):
             kb_path = kb_path.parent
 
         # if community layouts are supported, get them
-        if "LAYOUTS" in rules_mk:
-            for layout in rules_mk["LAYOUTS"].split():
-                cl_path = Path.cwd() / "layouts" / "community" / layout
+        if "LAYOUTS" in rules:
+            for layout in rules["LAYOUTS"].split():
+                cl_path = Path('layouts/community') / layout
                 if cl_path.exists():
                     names = names.union([keymap for keymap in cl_path.iterdir() if (cl_path / keymap / "keymap.c").is_file()])
```

Comment:
```suggestion
                    names = names.union([keymap.name for keymap in cl_path.iterdir() if (keymap / "keymap.c").is_file() or (keymap / "keymap.json").is_file()])
```

Reply:
`names = names.union([keymap.name for keymap in keymaps_dir.iterdir() if (keymap / "keymap.c").is_file() or (keymap / "keymap.json").is_file()])` is for Line168, but Github does not allow me to comment on it.